packages/@okta/vuepress-site/docs/guides/customize-themes/index.md

```diff
@@ -15,14 +15,13 @@ Use the Brands API to customize the theme of your Okta org.
 **What you need**
 
 * [Okta Developer Edition organization](https://developer.okta.com/signup)\
-* Access to the Brands API. Contact [Okta support](https://support.okta.com/help)
 * [Postman](https://www.postman.com/downloads/)
 
 ---
 
 ## About theme customization of your Okta org
 
-Okta provides you with a lot of power to authenticate your users, and using the [Okta-hosted Sign-In Widget](/docs/concepts/hosted-vs-embedded/#okta-hosted-widget) gets you up and running quickly, without having to write much custom code or host the sign-in functionality yourself. However, the trade-off is that your [customization options](/docs/guides/custom-widget/) are more limited, and potentially tricky to administer (with the existing functionality you tend to need to set colors, icons, and so on, in multiple places).
+Okta provides you with a lot of power to authenticate your users, and using the [Okta-hosted Sign-In Widget](/docs/concepts/redirect-vs-embedded/#redirect-authentication) gets you up and running quickly, without having to write much custom code or host the sign-in functionality yourself. However, the trade-off is that your [customization options](/docs/guides/custom-widget/) are more limited, and potentially tricky to administer (with the existing functionality you tend to need to set colors, icons, and so on, in multiple places).
```

Comment:
```suggestion
Okta provides you with a lot of power to authenticate your users, and using the [Okta-hosted Sign-In Widget](/docs/concepts/redirect-vs-embedded/#redirect-authentication) gets you up and running quickly without having to write much custom code or host the sign-in functionality yourself. However, the trade-off is that your [customization options](/docs/guides/custom-widget/) are more limited and potentially tricky to administer, since the existing functionality that you typically need to set are colors, icons, and so on in multiple places.
```